http-server-netty/src/main/java/io/micronaut/http/server/netty/handler/PipeliningServerHandler.java

```diff
@@ -325,7 +325,7 @@ void readComplete() {
      * {@link FileRegion}.
      *
      * @param response The response
-     * @param body     The body, or {@code null} if there is no body
+     * @param body     The body, or {@code null} if there is nobody
```

Comment:
@alvarosanchez Fixed. 

No, I did it myself with the help of inspections to the idea. Yes, I made a mistake in this place, so it's worth checking my changes. But most of them are misspellings, commas and articles

Reply:
Yes, I started reviewing and the vast majority of changes made sense, but when I stumbled upon this one it made me think it could be auto-generated somehow.

Thank you very much for your contributions!